app/middleware/prerender.js

```diff
@@ -1,7 +1,10 @@
-const mapLink = require('../lib/mapLink');
+const links = require('../lib/postcodeLinks');
 
 function results(req, res, next) {
-  res.locals.services = mapLink.addUrl(res.locals.postcodeLocationDetails, res.locals.services);
+  res.locals.services =
+    links.addMapUrl(res.locals.postcodeLocationDetails, res.locals.services);
+  res.locals.resultsInternalLink =
```

Comment:
Should this property be `resultsExternalUrl` given the function call is `addExternalUrl`?